dali/pipeline/executor/lowered_graph.cc

```diff
@@ -468,100 +468,6 @@ OpNode *OpGraph::NodePtr(std::string_view instance_name) {
   return nullptr;
 }
 
-namespace {
```

Comment:
This part is now done with the new graph, in graph2dot.h/cc. The new code is very similar - the logic is unchanged, node ids were removed (because there are no node ids in the new graph).